test/Elastic.Apm.Tests/SerializationTests.cs

```diff
@@ -25,8 +25,7 @@ public void StringTruncationTest()
 		{
 			var str = new string('a', 1200);
 
-			var transaction = new Transaction(new TestAgentComponents(), str, "test") { Result = "fail" };
-			transaction.End(/* duration: */ 1);
```

Comment:
Did we not have a `.End()` call here previously? Maybe it's still worth calling `.End()` before we serialize it, just to make sure that the duration is not changed by the `.End()` call.

Reply:
Fixed

Reply:
I've added another test (since this one is called `StringTruncationTest` so it is not related to `Duration`) to verify that `Duration` if already has value then is not changed by `End()`  - `Elastic.Apm.Tests.ApiTests.ApiTests.CallToEndDoesNotChangeAlreadySetDuration`